packages/worker/src/tests/TestConfiguration.ts

```diff
@@ -129,10 +160,19 @@ class TestConfiguration {
 
   // TENANCY
 
-  createTenant = async (): Promise<User> => {
+  createTenant = async ({
+    addTenantToGlobalDb,
+  }: {
+    addTenantToGlobalDb: boolean
```

Comment:
Default to `true`? It should be a fairly niche case where you don't want to do this

Reply:
I needed to add this because of this issue: https://github.com/Budibase/budibase/issues/6134
When running [these tests](https://github.com/Budibase/budibase/blob/68d147b0f75e0746b5c7347bb81ba87063a1cd6a/packages/worker/src/middleware/tests/tenancy.spec.ts) in parallel with the rest, we kept getting conflicts.
I think this might not be an issue anymore as we handle the 412 in the db level. I will double check it

Reply:
Yup, not needed anymore. This might have solve the issue as well